changelog/undistributed.rst

```diff
@@ -10,6 +10,9 @@
 --------------------------------------------------------------------------------
 
 * IOSXE
+    * added showstackwisevirtual
```

Comment:
Please remove this `showstackwisevirtual`, only add the change related to this pull request

Reply:
@LenzDong, This was removed.